apps/meteor/server/features/EmailInbox/EmailInbox_Outgoing.ts

```diff
@@ -151,26 +152,34 @@ slashCommands.add({
 });
 
 callbacks.add(
-	'beforeSaveMessage',
+	'afterSaveMessage',
 	function (message: IMessage, room: any) {
 		if (!room?.email?.inbox) {
 			return message;
 		}
 
-		if (message.file) {
-			message.attachments = message.attachments || [];
-			message.attachments.push({
-				actions: [
-					{
-						type: 'button',
-						text: t('Send_via_Email_as_attachment'),
-						msg: `/sendEmailAttachment ${message._id}`,
-						msg_in_chat_window: true,
-						msg_processing_type: 'sendMessage',
-					},
-				],
-			});
-
+		if (message.files?.length && message.u.username !== 'rocket.cat') {
```

Comment:
So, instead of editing the original message 🤔 this will send a 2nd message. Any chance this ends up in a loop? Since, the message will trigger afterSaveMessage again 👀

Reply:
well, this line breaks the loop that happened, as you said, the message.u.username !== 'rocket.cat' part prevents a response to the response.